src/internal/version/restore.go

```diff
@@ -0,0 +1,20 @@
+package version
+
+import "math"
+
+const (
+	// Restore1DataAndMetaFiles is the corso backup format version
+	// in which we split from storing just the data to storing both
+	// the data and metadata in two files.
+	Restore1DataAndMetaFiles = 1
+	// Restore2IsMetaMarker is a small improvement on
+	// VersionWithDataAndMetaFiles, but has a marker IsMeta which
+	// specifies if the file is a meta file or a data file.
+	Restore2IsMetaMarker = 3
```

Comment:
```suggestion
	Restore3IsMetaMarker = 3
```

`RestoreCurrNameInMeta` is technically 2, but holding off for now.